website/docs/advanced-concepts/granular-access-control/reference/default-roles.md

```diff
@@ -0,0 +1,663 @@
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td></td>
+                    <td></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                </tr>
+                <tr>
+                    <td>&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;Default Roles</td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td></td>
+                    <td></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                </tr>
+                <tr>
+                    <td>&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;Custom Roles</td>
+                    <td></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td></td>
+                    <td></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                </tr>
+                <tr>
+                    <td>**Workspace**</td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                </tr>
+                <tr>
+                    <td>**Audit Logs**</td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td><span style={{color: 'green', fontWeight: 'bold', fontSize: '16px'}}>(✓)</span></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+
+For more information about each permission, see [Permissions](/advanced-concepts/granular-access-control/reference/permissions).
+
+### Limitations
+
+Despite having extensive permissions, the Instance Administrator role in Appsmith cannot perform the following tasks:
+
+- Inability to access or update individual workspaces or app content. This means that Instance Administrators cannot view or edit the specific content within each workspace or app, such as queries, datasources, or their configurations. 
```

Comment:
I couldn't understand the difference between the two differences. They both sound the same to me.

Reply:
Yes, that was repeated. Fixed it now.